.env.template

```diff
@@ -0,0 +1,13 @@
+ZULIP_API_KEY=<your-zulip-api-key>
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<email-address>
+EMAIL_PASS=<your-Password>
```

Comment:
Email pass is not an env variable, if you are making a new env variable please update it in settings.py and README.

Reply:
Ohh, instead of pass I have to write password